beacon_node/network/src/beacon_processor/work_reprocessing_queue.rs

```diff
@@ -383,7 +383,12 @@ pub fn spawn_reprocess_scheduler<T: BeaconChainTypes>(
 ) -> Sender<ReprocessQueueMessage<T>> {
     let (work_reprocessing_tx, work_reprocessing_rx) = mpsc::channel(MAX_SCHEDULED_WORK_QUEUE_LEN);
     // Basic sanity check.
-    assert!(ADDITIONAL_QUEUED_BLOCK_DELAY < MAXIMUM_GOSSIP_CLOCK_DISPARITY);
+    assert!(
+        ADDITIONAL_QUEUED_BLOCK_DELAY
+            < Duration::from_millis(
+                T::EthSpec::default_spec().maximum_gossip_clock_disparity_millis
```

Comment:
this one can't be the default spec, needs to be threaded down until you get it here, probably via the `beacon_chain`